### ISSUE TITLE
Export to PDF

### ISSUE THREAD

**Issue by philipplozovoy** [yblake/deltaViz#7]
Hi, Yves Blake.
Is it possible to add properties to export this extension to pdf?
I have tried to do this by myself with this manual https://help.qlik.com/en-US/sense-developer/3.1/Subsystems/Extensions/Content/Howtos/extensions-enable-export.htm , but have no luck.
Maybe you could add ability to export this extension with export sheet to pdf function?

**Comment by philipplozovoy** [comment]
Now it is working.
I have replaced background.png  with absolute transparent picture.

**Comment by yblake** [comment]
Export support has been added in release 1.4.
Can you pls describe the exact issue ? any error message found in QS logs ?
Would you share you fix ?
Regards

**Comment by philipplozovoy** [comment]
Sorry for my English.

Issue
QlikSense function "Export to PDF" didn't work correctly with deltaViz extention. This function returned blank pdf instead of deltaViz chart.
I tried to add a property "support : { export: true} " and inside paint function add "return qlik.Promise.resolve();". It didn't help me with the issue.

Fix
I changed a file " backgroud.png" with this one (https://www.dropbox.com/s/0g339ojoio0mkso/background.png?dl=0).
